app/javascript/packs/application.js

```diff
@@ -7,6 +7,10 @@ import Rails from "@rails/ujs"
 import Turbolinks from "turbolinks"
 import * as ActiveStorage from "@rails/activestorage"
 import "channels"
+import "bootstrap"
+import "../stylesheets/application"
+import "./custom"
```

Comment:
try to avoid such ambiguous names. when the application is growing, it's impossible to predict what happens in files like that, and also keeping everything in mind is an inadequate requirement to deal with anything

suppose `effects` or sth similar would be good to start with